data/json/monsterdrops/zombie_survivor.json

```diff
@@ -9,10 +9,47 @@
       { "distribution": [
         { "group": "book_survival", "prob": 25 },
         { "group": "gear_survival", "prob": 25, "damage": [ 1, 4 ] },
-        { "group": "guns_survival", "prob": 25, "damage": [ 1, 4 ] },
+        { "group": "survivor_weapons", "prob": 50, "damage": [ 1, 4 ] },
+        { "group": "survivor_grenades", "prob": 15, "damage": [ 1, 4 ] },
         { "group": "stash_food", "prob": 40 }
       ] },
       { "item": "cash_card" }
     ]
+  },
+  {
+    "id" : "survivor_melee",
+    "type" : "item_group",
+    "items": [
+      [ "ax", 5 ],
+      [ "bwirebat",15 ],
+      [ "crowbar", 5 ],
+      [ "nailboard", 15 ],
+      [ "machete", 5 ],
+      [ "makeshift_crowbar", 25 ],
+      [ "makeshift_knife", 20 ],
```

Comment:
Could also have copper knife, copper spear and maybe wooden spear. Those are relatively easy to craft, but quite good.